cli/gluster-block.c

```diff
@@ -390,23 +392,48 @@ glusterBlockIsAddrAcceptable(char *addr)
     if (!isdigit(addr[i]) && (addr[i] != '.'))
       return FALSE;
   }
+
+  if (!inet_pton(AF_INET, addr, &tmp) && !inet_pton(AF_INET6, addr, &tmp)) {
+    return FALSE;
```

Comment:
```
       AF_INET6
              src points to a character string containing an IPv6 network address.  The address is converted to a struct in6_addr and  copied  to  dst,  which  must  be
              sizeof(struct in6_addr) (16) bytes (128 bits) long.  The allowed formats for IPv6 addresses follow these rules:
```

For the size of "tmp", for the AF_INET6 case it may crash ?

Reply:
@lxbsz thanks, fixed it now.